projects/dotcms-field-elements/src/components/dot-textarea/dot-textarea.tsx

```diff
@@ -65,9 +65,10 @@ export class DotTextareaComponent {
     render() {
         return (
             <Fragment>
-                <label>{this.label}</label>
+                <label htmlFor={this.name}>{this.label}</label>
                 <textarea
                     class={this.getInputClassName()}
+                    id={this.name}
```

Comment:
the field name is not unique should be the variable

Reply:
![image](https://user-images.githubusercontent.com/37185433/56523206-6d53e500-6504-11e9-9859-efd674105410.png)

we are currently using `variable` param